### PR TITLE
Closes #43

### LINKED ISSUE #43: Create a dataset loader for MeDAL
From https://github.com/BruceWen120/medal

### PR DESCRIPTION
**Pull request by alisoncallahan**
- **Name:** *MeDAL*
- **Description:** *Repository for Medical Dataset for Abbreviation Disambiguation for Natural Language Understanding (MeDAL), a large medical text dataset curated for abbreviation disambiguation, designed for natural language understanding pre-training in the medical domain. It was published at the ClinicalNLP workshop at EMNLP.*
- **Paper:** *https://arxiv.org/abs/2012.13978*
- **Data:** *https://zenodo.org/record/4482922#.YlmqP5PMKrM*

### Checkbox

- [x] Confirm that this PR is linked to the dataset issue.
- [x] Create the dataloader script `biodatasets/my_dataset/my_dataset.py` (please use only lowercase and underscore for dataset naming).
- [x] Provide values for the `_CITATION`, `_DATASETNAME`, `_DESCRIPTION`, `_HOMEPAGE`, `_LICENSE`, `_URLs`, `_SUPPORTED_TASKS`, `_SOURCE_VERSION`, and `_BIGBIO_VERSION` variables.
- [x] Implement `_info()`, `_split_generators()` and `_generate_examples()` in dataloader script.
- [x] Make sure that the `BUILDER_CONFIGS` class attribute is a list with at least one `BigBioConfig` for the source schema and one for a bigbio schema.
- [x] Confirm dataloader script works with `datasets.load_dataset` function.
- [x] Confirm that your dataloader script passes the test suite run with `python -m tests.test_bigbio biodatasets/my_dataset/my_dataset.py`.
- [ ] If my dataset is local, I have provided an output of the unit-tests in the PR (please copy paste). This is OPTIONAL for public datasets, as we can test these without access to the data files.

### CODE REVIEW

File: biodatasets/medal/medal.py [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+    def _info(self) -> datasets.DatasetInfo:
+
+        # Create the source schema; this schema will keep all keys/information/labels as close to the original dataset as possible.
+
+        # You can arbitrarily nest lists and dictionaries.
+        # For iterables, use lists over tuples or `datasets.Sequence`
+
+        if self.config.schema == "source":
+            features=datasets.Features(
```
Comment (jason-fries) [inline biodatasets/medal/medal.py:134]: Remove the trailing comma :) 

```
        if self.config.schema == "source":
            features = datasets.Features(
                {
                    "abstract_id": datasets.Value("int32"),
                    "text": datasets.Value("string"),
                    "location": datasets.Sequence(datasets.Value("int32")),
                    "label": datasets.Sequence(datasets.Value("string")),
                }
            )
```

**Review by jason-fries — CHANGES_REQUESTED** [review]

File: biodatasets/medal/medal.py (hunk 1 above)
Comment (alisoncallahan) [reply biodatasets/medal/medal.py:134]: I did, still the same issue.

Comment (alisoncallahan) [reply biodatasets/medal/medal.py:134]: Thanks! That fixed it.

**Comment by jason-fries** [comment]
This PR is getting thorny -- deleting so @alisoncallahan can just push a fresh one.

**Pull request closed without merging**